README.md

```diff
@@ -134,6 +134,14 @@ Get the request headers.
 
 Get the request body.
 
+#### .progress(loaded : number, total : number, lengthComputable : bool)
```

Comment:
That's a good suggestion.  I like the symmetry of having the upload related events on the request and the download related events on the response.